### DISCUSSION TITLE
plugins in Windows exe

### DISCUSSION THREAD

**Discussion by crass00** [comictagger/comictagger#606] [Ideas]
Hi,

Since us windows users (afaik) can't use the external metadata talkers,  would it be possible to include the known talkers from mizaki to the compiled Windows/MacOS applications

Thanks

**Comment by lordwelch** [comment]
1.6.0-alpha.11 is broken but 10 should let you use metron and if 11 worked it has GCD as well

**Comment by lordwelch** [comment]
@crass00 You can now download the whl or zip files from their release pages and install it in the plugin directory, see the [wiki] for more info

[wiki]: https://github.com/comictagger/comictagger/wiki/Installing-plugins
